crates/orderbook/src/arguments.rs

```diff
@@ -76,6 +76,10 @@ pub struct Arguments {
     #[clap(long, env, default_value = "200")]
     pub pool_cache_lru_size: usize,
 
+    /// Enable EIP-1271 orders.
+    #[clap(long, env, parse(try_from_str), default_value = "false")]
```

Comment:
Isn't this required because we want `--enable-eip1271-orders true` instead of `--enable-eip1271-order` flag?

Note we do the same thing for the `--enable-presign-orders` option.